packages/geo/src/lib/measure/measurer/measurer.component.ts

```diff
@@ -380,33 +383,9 @@ export class MeasurerComponent implements OnInit, OnDestroy {
   private initStore() {
     const store = this.store;
 
-    if (store.layer === undefined) {
-      const layer = new VectorLayer({
-        title: 'Measures',
-        zIndex: 200,
-        source: new FeatureDataSource(),
-        style: createMeasureLayerStyle(),
-        showInLayerList: false
-      });
-      store.bindLayer(layer);
-    }
-
-    if (store.layer.map === undefined) {
-      this.map.addLayer(store.layer);
-    }
-
-    if (store.getStrategyOfType(FeatureStoreLoadingStrategy) === undefined) {
-      store.addStrategy(new FeatureStoreLoadingStrategy({}));
-    }
-    store.activateStrategyOfType(FeatureStoreLoadingStrategy);
-
-    if (store.getStrategyOfType(FeatureStoreSelectionStrategy) === undefined) {
-      store.addStrategy(new FeatureStoreSelectionStrategy({
-        map: this.map,
-        many: true
-      }));
-    }
-    store.activateStrategyOfType(FeatureStoreSelectionStrategy);
+    tryBindStoreLayer(store);
```

Comment:
It appears that you completely replace the layer by the basic layer created by that function. Create hte layer first then pass it to the function.